flavors/lineageos/device-metadata.json

```diff
@@ -1,11 +1,4 @@
 {
-  "Amber": {
-    "branch": "lineage-17.1",
-    "lineage_recovery": false,
-    "name": "Phone",
-    "variant": "userdebug",
-    "vendor": "yandex"
-  },
```

Comment:
LOS 17.1 is deprecated now. I'd like to keep it the code around though for those wanting to build for legacy devices on their own.